metadata-ingestion/src/datahub/ingestion/source/metabase.py

```diff
@@ -441,7 +441,7 @@ def get_datasource_urn(self, card_details):
             card_details.get("database_id", "")
         )
         query_type = card_details.get("dataset_query", {}).get("type", {})
-        source_paths = set()
+        source_tables = set()
```

Comment:
`source_paths` was a set of `schema.table` strings.
`source_tables` is a set of fully qualified `database.schema.table`  strings, moving some existing logic around to get the database name for "native" queries (`query_type == "query"`) and adding new logic to get the database name for SQL queries (`query_type != "query"`).